client/my-sites/checkout/composite-checkout/components/checkout-sidebar-plan-upsell/index.tsx

```diff
@@ -94,8 +95,10 @@ export function CheckoutSidebarPlanUpsell() {
 		{ strong: createElement( 'strong' ) }
 	);
 
+	const isEnglish = ( config( 'english_locales' ) as string[] ).includes( locale || '' );
+
 	if (
-		'en' !== locale &&
+		! isEnglish &&
 		! hasTranslation( '<strong>Save %(percentSavings)d%%</strong> by paying for two years' )
```

Comment:
Now that [this job](https://translate.wordpress.com/deliverables/overview/7606775/) is finished, is there any reason we shouldn't remove the guard entirely?

Reply:
Thanks for the reviews @sirbrillig and @michaeldcain . This suggestion makes sense to me, so I've gone ahead and removed this check. Could you please check again? Thanks.